static/install/web.html

```diff
@@ -28,7 +28,7 @@
         <link rel="license" href="/LICENSE.txt"/>
         <script type="module" src="/js/redirect.js"></script>
         <script type="module" src="/js/fastboot/dist/fastboot.min.mjs?1.0.8"></script>
-        <script type="module" src="/js/web-install.js"></script>
+        <script type="module" src="/js/web-install.js?1"></script>
```

Comment:
Should no longer add the cache busting strings. It's handled automatically now for `/js/*.js` but not the fastboot library.